libcxx/include/__vector/vector.h

```diff
@@ -775,6 +780,17 @@ class _LIBCPP_TEMPLATE_VIS vector {
   }
 
   _LIBCPP_CONSTEXPR_SINCE_CXX20 _LIBCPP_HIDE_FROM_ABI void __move_assign_alloc(vector&, false_type) _NOEXCEPT {}
+
+  static _LIBCPP_CONSTEXPR_SINCE_CXX20 _LIBCPP_HIDE_FROM_ABI pointer __add_alignment_assumption(pointer __p) _NOEXCEPT {
```

Comment:
Similarly to https://github.com/llvm/llvm-project/pull/118837#pullrequestreview-2578321015 I think this one needs `_LIBCPP_NO_CFI` since we're potentially static_cast'ing uninitialized memory (for the end pointer).

(Apologies for the late notice, we're a bit behind on libc++.)

Reply:
Ack. I'd be curious to see how this failure can be reproduced. Is it just a matter of adding a `-fsanitize=cfi` job to our pre-commit CI?

Reply:
Also https://github.com/llvm/llvm-project/pull/124839 for adding NO_CFI

Reply:
I haven't tried to reproduce it on the libc++ tests directly, but it should be possible. I think we're currently only using a subset of the cfi checks: `-fsanitize=cfi-vcall -fsanitize=cfi-derived-cast -fsanitize=cfi-unrelated-cast`. The annoying thing is that cfi requires (thin) lto.